### PR TITLE
Updated CLI to accept --root

### PR DESCRIPTION
**Pull request by walkingriver**
I saw that the API version of this command lets you specify the root URL in code, but not at the command line. This PR addresses that, adding --root (-r and WSD_ROOT) as CLI options.

### CODE REVIEW

File: lib/wsd.js [hunk 1]
```diff
@@ -91,18 +92,18 @@ class WSD {
    *   formats include: 'png', 'svg', or 'pdf'.  'pdf' requires a paid
    *   account.
    * @param {string} [apikey] API key for non-free usage.
+   * @param {string} [root] Root URL for the service.
```
Comment (hildjj) [inline lib/wsd.js:95]: `[root='https://websequencediagrams.com']`

**Review by hildjj — APPROVED** [review]
Minor nit, if you want to fix it before I merge.  Thanks for the change!

Once that's in, I'll update dependencies, update action versions in GHA, then do a release.

File: lib/wsd.js (hunk 1 above)
Comment (walkingriver) [reply lib/wsd.js:95]: I’ll be out for the next few hours, so if you can make that change, please do. Thanks!

**Comment by walkingriver** [comment]
Do we need to bump its version number to 2.1.x or something? I'm not seeing a new version when I try to update it via `npm`.

**Comment by hildjj** [comment]
I haven't published yet, sorry. I think we need to do a major version bump because I removed support for node 12. Wanted to look at it again with fresh eyes before I pulled the trigger. 

Everything else look ok to you in HEAD?

**Comment by walkingriver** [comment]
I haven't seen any issues, no. I do agree that a major bump is probably warranted, given that you are dropping support for Node 12.

**Comment by hildjj** [comment]
Released v3.0.0

**Pull request merged**